subsys/net/lib/http/Kconfig

```diff
@@ -39,9 +39,11 @@ config HTTP_HEADERS
 	int "HTTP header field max number of items"
 	depends on HTTP_SERVER
 	default 8
+	default 20 if WEBSOCKET
```

Comment:
Is such a high value based on something? My visual memory says that no. of headers in websocket establishment request was under a dozen.

Reply:
For example Firefox sends lot of headers, we need to be prepared to receive all of them as otherwise the handshake will fail.